README.md

```diff
@@ -29,3 +29,8 @@ What is the asymptotic runtime ($\Theta$) of your algorithm in terms of the
 number of comparisons of pancakes? What is it in terms of the number of flips?
 Add your answer to this markdown file.
 
+The asymptotic runtime in terms of comparisons will be n(n-k). The first n is the first outer loop, n-k is the second loop where k is the spaces left over after finding the right spot. $n(n-k) \in \Theta (n^2)$ Since k is a constant.
```

Comment:
Is $k$ a constant? Doesn't the number of positions not traversed differ based on which pancake you're moving into position?